module-2/backend/week_one.md

```diff
@@ -5,30 +5,84 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET - used to grab an object from the database

```

Comment:
Not necessarily an object from the database. This is used to get a view from our server. Frequently this requires our app to pull something from the database as well, but not always (think of a welcome page, or a `new` form).